wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -309,6 +309,16 @@ class Command : public wpi::Sendable, public wpi::SendableHelper<Command> {
   [[nodiscard]]
   CommandPtr OnlyIf(std::function<bool()> condition) &&;
 
+  /**
+   * Creates a new command that runs this command
+   * and the deadline in parallel, finishing when
+   * the deadline finishes.
+   *
+   * @param deadline the deadline of the command group
+   * @return the decorated command
+   */
```

Comment:
Open question: Do we want to include an explicit note that the calling command will be interrupted when the other command finishes? (This would make it match more with `DeadlineFor`)
Also, do we want to add `@see DeadlineFor` to the `WithDeadline()` doc comment and `@see WithDeadline` to the `DeadlineFor` doc comment?

(Both of these apply to Java, by the way)

Reply:
makes sense; i'll get around to it soon